lib/Models/ArcGisFeatureServerCatalogItem.js

```diff
@@ -108,10 +119,65 @@ ArcGisFeatureServerCatalogItem.prototype._getValuesThatInfluenceLoad = function(
 ArcGisFeatureServerCatalogItem.prototype._load = function() {
   var that = this;
   this._geoJsonItem = new GeoJsonCatalogItem(this.terria);
-  this._geoJsonItem.data = loadGeoJson(this);
-  return that._geoJsonItem.load().then(function() {
-    that.rectangle = that._geoJsonItem.rectangle;
-  });
+  that._geoJsonItem.data = loadGeoJson(that);
+
+  if (this.inheritStyleInformation) {
+    return loadMetadata(this).then(function(val) {
+      that._metadata = val;
+
+      const renderer = that._metadata.drawingInfo.renderer;
+
+      if (!defined(that._geoJsonItem.style)) that._geoJsonItem.style = {};
+
+      // A 'simple' renderer only applies a single style to all features
+      if (that._metadata.drawingInfo.renderer.type === "simple") {
+        const style = convertEsriRendererToStyle(renderer.symbol);
+        const geojsonStyle = that._geoJsonItem.style;
+        Object.assign(geojsonStyle, style);
+      }
+
+      return that._geoJsonItem.load().then(function() {
+        that.rectangle = that._geoJsonItem.rectangle;
+
+        // For a 'uniqueValue' renderer symbology gets applied via feature properties.
+        if (renderer.type === "uniqueValue") {
+          const rendererObj = setupUniqueValRenderer(renderer);
+
+          const primaryFieldForSymbology = renderer.field1;
+
+          that._geoJsonItem.data.features.forEach(function(f) {
+            let symbolName = f.properties[primaryFieldForSymbology];
+
+            // accumulate values if there is more than one field defined
+            if (renderer.fieldDelimiter && renderer.field2) {
+              var val2 = f.properties[renderer.field2];
+              if (val2) {
+                symbolName += renderer.fieldDelimiter + val2;
+                var val3 = f.properties[renderer.field3];
+                if (val3) {
+                  symbolName += renderer.fieldDelimiter + val3;
+                }
+              }
+            }
+
+            let rendererStyle = rendererObj[symbolName];
+            if (rendererStyle === null)
+              rendererStyle = that._metadata.drawingInfo.renderer.defaultSymbol;
+            Object.assign(
```

Comment:
Make sure `Object.assign` is ok in IE11. It used to not be, but it might be ok now. If it's still a problem, you can use Cesium's `combine` function instead.

Reply:
You are right - and I feel like you knew that that wouldn't work!

Reply:
Haha I really wasn't sure. We've adjusted our polyfills enough times now that I've lost track. I think in mobx it would be ok now, at least.